<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>780</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="button_pon">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>90</horstretch>
             <verstretch>90</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>90</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>90</width>
             <height>90</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Пн</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_vtor">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>90</horstretch>
             <verstretch>90</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>90</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Вт</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_sreda">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>90</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Ср</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_chetv">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>90</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Чт</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_pyatn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>90</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Пт</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_subb">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>90</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Сб</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_vsk">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>90</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Вс</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="Month">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>MS Shell Dlg 2</family>
             <pointsize>16</pointsize>
             <weight>9</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="statusTip">
            <string/>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>month</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_Day">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Day</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>24</pointsize>
           <weight>9</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 24pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Планы на сегодня</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_addTask">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 50px;
font: 75 30pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 0, 0);
</string>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>771</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="task_1">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="task_2">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="task_3">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1159</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
